crates/ruff_workspace/src/options.rs

```diff
@@ -2397,9 +2397,26 @@ pub struct PydocstyleOptions {
     /// convention = "google"
     /// ```
     ///
-    /// As conventions force-disable all rules not included in the convention,
-    /// enabling _additional_ rules on top of a convention is currently
-    /// unsupported.
+    /// If you must, you can enable rules not included in a convention by
```

Comment:
Is there another place I should update the docs, or is this the right place?

Reply:
This is the right place :)